src/composables/fungibleTokens.ts

```diff
@@ -253,10 +254,11 @@ export function useFungibleTokens() {
 
     const tokenData = protocol
       && getProtocolAvailableTokens(protocol)[contractCallData?.assetContractId!];
+
     if (contractCallData && tokenData) {
       return +toShiftedBigNumber(
         contractCallData.amount || 0,
-        -tokenData.decimals!,
+        -(tokenData.decimals || AE_COIN_PRECISION),
```

Comment:
This point is pretty moot. In most cases it will be unnoticeable (since the base amount of decimals is 18 in ae fungible tokens). But in cases when it is not, the user will think that he lost money, or it was stolen.
In case we use zero as fallback it will be obvious to the user that it is an incorrect balance, but it will be highly noticeable in most cases.

Reply:
Agreed on priv that we leave 18 for now.